requirements.txt

```diff
@@ -1,10 +1,10 @@
-kubernetes==9.0.1
-jupyterhub==1.1.0
-jupyterhub-kubespawner==0.12.0
+kubernetes==10.1.0
+jupyterhub==1.2.1
+git+https://github.com/jupyterhub/oauthenticator.git@15fa131853a047eb8acdfddfeb43da738812ed70
+jupyterhub-kubespawner==0.14.1
 git+https://github.com/jupyterhub/wrapspawner.git@5f2b7075f77d0c1c49066682a8e8adad0dab76db
 jupyterhub-tmpauthenticator==0.6
-oauthenticator==0.9.0
 jupyterhub-ldapauthenticator==1.2.2
-psycopg2==2.8.4
+psycopg2==2.8.6
 openshift==0.10.0
```

Comment:
Seems like we should use newer version of OpenShift to match requirement of JH ODH https://github.com/opendatahub-io/jupyterhub-odh/blob/master/Pipfile#L9

Reply:
updated to 0.11.2